clientv3/snapshot/util.go

```diff
@@ -33,3 +36,4 @@ func bytesToRev(bytes []byte) revision {
 type initIndex int
 
 func (i *initIndex) ConsistentIndex() uint64 { return uint64(*i) }
+func (i *initIndex) Save(backend.BatchTx)    {}
```

Comment:
I am not sure if this affects the snapshot restore. As the result of this, the following line won't actually save the consistent index to disk?
https://github.com/etcd-io/etcd/blob/2da44a9ddfa9e037848f1e31a287936fa25e58b3/clientv3/snapshot/v3_snapshot.go#L402

Reply:
thanks. it does not look good, and i will  give a better solution.

Reply:
Thanks for digging into this. I am not quite familiar with this part of the code. Just want to make sure snapshot restore behaves the same.

Reply:
i have cleaned up initIndex and verified that snapshot restore behaves the same.